evap/rewards/tests/test_tools.py

```diff
@@ -50,3 +51,22 @@ def test_already_got_points(self):
         mommy.make(RewardPointGranting, user_profile=self.student, value=0, semester=self.course.semester)
         self.form.submit()
         self.assertEqual(0, reward_points_of_user(self.student))
+
+class TestGrantRewardPointsParticipationChange(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.course = mommy.make(Course)
+        already_evaluated = mommy.make(Course, semester=cls.course.semester)
+        SemesterActivation.objects.create(semester=cls.course.semester, is_active=True)
+        cls.student = mommy.make(UserProfile, username="student", email="foo@institution.example.com",
+            courses_participating_in=[cls.course, already_evaluated], courses_voted_for=[already_evaluated])
+
+    def test_participant_removed_from_course(self):
+        self.course.participants.remove(self.student)
+
+        self.assertEqual(settings.REWARD_POINTS_PER_SEMESTER, reward_points_of_user(self.student))
+
+    def test_course_removed_from_participant(self):
+        self.student.courses_participating_in.remove(self.course)
+
+        self.assertEqual(settings.REWARD_POINTS_PER_SEMESTER, reward_points_of_user(self.student))
```

Comment:
i would say the variable is equal to the constant, and not the constant is equal to the variable, if you know what i mean.